src/Utils/CMakeLists.txt

```diff
@@ -16,16 +20,27 @@ set(hiopUtils_INTERFACE_HEADERS
   hiopOptions.hpp
   hiopRunStats.hpp
   hiopTimer.hpp
-  )
+)
+
+if(HIOP_USE_RAJA)
+  list(APPEND hiopUtils_INTERFACE_HEADERS RajaUmpireUtils.hpp)
+  list(APPEND hiopUtils_SRC ${hiopUtils_RAJA_SRC})
+endif()
+
+if(HIOP_USE_CUDA)
+  set_source_files_properties(${hiopUtils_RAJA_SRC} PROPERTIES LANGUAGE CUDA)
+endif()
+if(HIOP_USE_HIP)
+  #this needs to be checked with hip/rocm
+  set_source_files_properties(${hiopUtils_RAJA_SRC} PROPERTIES LANGUAGE CXX)
+  set_source_files_properties(${hiopUtils_RAJA_SRC} PROPERTIES COMPILE_FLAGS "-x hip")
```

Comment:
This is the recommended way by the ORNL training for that particular platform.
It might be outdated since `cmake` may have already added support to `set_source_files_properties(xxxx PROPERTIES LANGUAGE HIP)

Reply:
I guess, both of us ran into https://www.olcf.ornl.gov/wp-content/uploads/2021/01/2021UM-Day-3-Joo-Developing-for-Frontier-using-HIP-on-Spock.pdf

Reply:
yes! Good to know that this doc is on public domain.